Add processing and notification responses

caleeli opened the pull request.

velkymx commented on ProcessMaker/Jobs/InstanceAction.php:
```diff
@@ -0,0 +1,51 @@
+            //Run engine to the next state
+            $workflow->getEngine()->runToNextState();
+        } catch (\Throwable $t) {
+            dd($t);
```
Move to Laravel log

velkymx commented on ProcessMaker/Jobs/StartEvent.php:
```diff
@@ -0,0 +1,77 @@
+            
+            return $response;
+        } catch (\Throwable $t) {
+            dd($t);
```
Log

velkymx commented on ProcessMaker/Jobs/TokenAction.php:
```diff
@@ -0,0 +1,70 @@
+            //Run engine to the next state
+            $workflow->getEngine()->runToNextState();
+        } catch (\Throwable $t) {
+            dd($t);
```
Log

velkymx commented on ProcessMaker/Repositories/ExecutionInstanceRepository.php:
```diff
@@ -0,0 +1,132 @@
+    {
+        $instance = Instance::where('uid', $uid)->first();
+        if (!$instance)
+            return;
```
Is there a better way to return?

velkymx commented on resources/views/layouts/layout.blade.php:
```diff
@@ -6,7 +6,15 @@
     <meta name="user-uid" content="{{\Auth::user()->uid}}">
     <meta name="csrf-token" content="{{ csrf_token() }}">
+    @if(Auth::user())
+    <meta name="user-uid" content="{{ Auth::user()->uid }}">
```
Duplicate line

velkymx commented on resources/views/layouts/layout.blade.php:
```diff
@@ -6,7 +6,15 @@
+    <meta name="user-id" content="{{ Auth::user()->id }}">
+    @endif
     <meta name="api-token" content="{{ session('apiToken')['access_token']}}">
+    <meta name="broadcaster" content="{{config('broadcasting.broadcaster')}}">
```
Move broadcaster meta into @if(config('broadcasting.broadcaster') block

caleeli commented:
- Added Laravel Log to register fails during the flow execution.
- Trigger a 404 exception when a instance does not exists in the DB
- Add validation to load socket.io parameters in the meta headers
- Updated composer.lock to the latest Nayra revision.
